src/lightning_lite/plugins/precision/native_amp.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         if precision == "bf16" and not _TORCH_GREATER_EQUAL_1_10:
             raise ImportError("To use bfloat16 with native amp you must install torch greater or equal to 1.10.")
         if scaler is None and precision == 16:
-            with _patch_cuda_is_available():
+            with _patch_cuda_is_available():  # TODO: Remove once minimum supported PyTorch version is 1.14
```

Comment:
The implementation already has this comment. No need to add it again here

```suggestion
            with _patch_cuda_is_available():
```